web/src/recent_view_ui.ts

```diff
@@ -287,9 +301,9 @@ function set_table_focus(row, col, using_keyboard) {
     $current_focus_elem = "table";
 
     if (using_keyboard) {
-        const scroll_element = document.querySelector(
+        const scroll_element = $(
             "#recent_view_table .table_fix_head .simplebar-content-wrapper",
-        );
+        )[0];
```

Comment:
@amanagr do you remember why we're using `document.querySelector` here? I could imagine a perf story for avoiding jQuery.

Reply:
Yeah, mainly because we don't the jQuery selector here and using `document.querySelector` is faster. If it makes writing Typescript easier, I don't have any issues with switching it to use jQuery.

Reply:
Yeah, the two issues I encountered were that `scroll_element` could be `null` (easily fixed with an assert) and `Property 'offsetHeight' does not exist on type 'Element'.ts(2339)` which I didn't know how to address without using JQuery instead.

Reply:
You just need it to be declared as an `HTMLElement`, which is probably possible but we haven't figured out the pattern for that with `.querySelector`